test-values.yaml

```diff
@@ -103,45 +103,48 @@ initContainers:
           mountPath: "/etc/trino/generated"
 
 accessControl:
-  type: configmap
-  refreshPeriod: 60s
-  configFile: "rules.json"
-  rules:
-    rules.json: |-
-      {
-        "catalogs": [
-          {
-            "user": "admin",
-            "catalog": "(tpch|system)",
-            "allow": "all"
-          },
-          {
-            "group": "group1",
-            "catalog": "tpch",
-            "allow": true
-          },
-          {
-            "user": "alice",
-            "catalog": "jmx",
-            "allow": "read-only"
-          },
-          {
-            "catalog": "system",
-            "allow": "none"
-          }
-        ],
-        "schemas": [
-          {
-            "user": "admin",
-            "schema": ".*",
-            "owner": true
-          },
-          {
-            "user": "alice",
-            "owner": false
-          }
-        ]
-      }
+  type: custom
+  customValues: |
+    access-control.name=read-only
+  # type: configmap
```

Comment:
Remove commented out code in tests. Add a separate test in `test.sh` so we can cover both cases.